sktime/forecasting/base/adapters/_fbprophet.py

```diff
@@ -114,19 +109,86 @@ def _predict(self, fh=None, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
             X = X.copy()
             df, X = _merge_X(df, X)
 
-        # don't compute confidence intervals if not asked for
-        with self._return_pred_int(return_pred_int):
-            out = self._forecaster.predict(df)
+        out = self._forecaster.predict(df)
 
         out.set_index("ds", inplace=True)
         y_pred = out.loc[:, "yhat"]
 
-        if return_pred_int:
+        y_pred = pd.DataFrame(y_pred)
+        y_pred.reset_index(inplace=True)
+        y_pred.index = y_pred["ds"].values
+        y_pred.drop("ds", axis=1, inplace=True)
+        return y_pred
+
+    def _predict_quantiles(self, fh, X=None, alpha=DEFAULT_ALPHA):
+        """
+        Compute/return prediction quantiles for a forecast.
+
+        Must be run *after* the forecaster has been fitted.
+
+        If alpha is iterable, multiple quantiles will be calculated.
+
+        Parameters
+        ----------
+        fh : int, list, np.array or ForecastingHorizon
+            Forecasting horizon, default = y.index (in-sample forecast)
+        X : pd.DataFrame, optional (default=None)
+            Exogenous time series
+        alpha : float or list of float, optional (default=[0.05, 0.95])
+            A probability or list of, at which quantile forecasts are computed.
+
+        Returns
+        -------
+        quantiles : pd.DataFrame
+            Column has multi-index: first level is variable name from y in fit,
+                second level being the values of alpha passed to the function.
+            Row index is fh. Entries are quantile forecasts, for var in col index,
+                at quantile probability in second col index, for the row index.
+        """
+        self.check_is_fitted()
```

Comment:
these two checks are already done in the "non-underscore" function, kindly remove